.eslintrc

```diff
@@ -0,0 +1,35 @@
+env:
+  node: true
+  browser: true
+
+rules:
+  brace-style: 2
+  comma-spacing: 1
+  eol-last: 0
+  indent: [2, 2]
+  key-spacing: 0
+  new-parens: 2
+  no-debugger: 2
+  no-dupe-args: 2
+  no-dupe-keys: 2
+  no-duplicate-case: 2
+  no-ex-assign: 2
+  no-extra-parens: 1
+  no-fallthrough: 2
+  no-multi-spaces: [2, { exceptions: { "ObjectExpression": true } }]
```

Comment:
```js
return self.sendMpdCommandArray([
  {command: 'clear', parameters: []},
  {command: 'add',   parameters: [arrayTrackUris.shift()]},
  {command: 'play',  parameters: []}
])
```
The middle and last line would be an error otherwise